root/tmp/setup/php-extensions.sh

```diff
@@ -2,16 +2,18 @@
 
 set -e
 
-echo "Installing apt depdencies"
+echo "Installing apt dependencies"
 
-BUILD_PACKAGES="gettext libcurl4-openssl-dev libpq-dev libmysqlclient-dev libldap2-dev libxslt-dev \
+BUILD_PACKAGES="gettext libcurl4-openssl-dev libpq-dev libmariadbclient-dev libldap2-dev libxslt-dev \
```

Comment:
Hi @andrewnicols,
what about alphabetically sorting the packages here and in `LIBS`/`RUNTIME`? (Applies to all PRs for #21)

Guessing that the initial purpose was kind of group-by per need, to satisfy requirements of those packages to be installed... but during the time such a purpose has been "blurred" so IMHO we have to options here:
a. sorting, w/ no ref to any single package requirement
b. grouping the needs using other vars to keep an eye on the requirements of each single package to be installed later - not really an advantage.

TIA,
Matteo